stdlib/list.gr

```diff
@@ -246,4 +246,14 @@ let findIndex = (fn, list) => {
     }
   }
   findItemIndex(list, 0)
-}
\ No newline at end of file
+}
+
+let product = (a, b) => {
+  let mut list = []
+  forEach((aItem) => {
+    forEach((bItem) => {
+     list = append(list, [(aItem, bItem)])
```

Comment:
Here we could instead do `list = [(aItem, bItem), ...list]` and call `reverse` at the end for a bit more efficiency 🙂

Reply:
@ospencer  good point 👍